packages/arb-token-bridge-ui/src/hooks/TransferPanel/useGasSummary.ts

```diff
@@ -103,109 +83,90 @@ export function useGasSummary(): UseGasSummaryResult {
     []
   )
 
-  const estimateGas = useCallback(async () => {
-    if (!walletAddress) {
+  const { gasEstimates: estimateGasResult, error: gasEstimatesError } =
+    useGasEstimates({
+      txType: isDepositMode ? 'deposit' : 'withdrawal',
+      walletAddress,
+      childChainProvider,
+      parentChainProvider: isDepositMode ? parentChainProvider : undefined,
```

Comment:
why undefined?

Reply:
deposit needs both but withdrawal only needs child chain provider

Reply:
you can still pass both